### ISSUE TITLE
Redis does not start

### ISSUE THREAD

**Issue by waldner** [graphite-project/docker-graphite-statsd#118]
Image: graphiteapp/graphite-statsd:1.1.5-12

Setting `REDIS_TAGDB=true` attemps to use redis for tagDB, but it fails because of the following error:

```
*** FATAL CONFIG FILE ERROR ***
Reading the configuration file, at line 8
>>> 'logfile /var/log/redis/redis-server.log'
Can't open the log file: No such file or directory
```
periodically repeated in the log, as runsv tries to restart it (I suppose). Since /var/log is host-mounted in my setup, I can create the directory by hand and then it works, but I believe it should not be necessary.

**Comment by deniszh** [comment]
Hi @waldner,

Sorry, can't reproduce.

```
% docker run -e "REDIS_TAGDB=true" \
 --name graphite\
 --restart=always\
 -p 80:80\
 -p 2003-2004:2003-2004\
 -p 2023-2024:2023-2024\
 -p 8125:8125/udp\
 -p 8126:8126\
 graphiteapp/graphite-statsd
Started runsvdir, PID is 40
wait for processes to start....
/usr/sbin/crond 4.5 dillon's cron daemon, started with loglevel notice
...
Superuser created successfully.
ok: run: nginx: (pid 51) 3s
run: carbon: (pid 57) 5s; run: log: (pid 56) 5s
run: carbon-aggregator: (pid 67) 5s; run: log: (pid 65) 5s
down: carbon-relay: 1s, normally up, want up; run: log: (pid 58) 5s
down: collectd: 1s, normally up, want up
run: cron: (pid 54) 5s
run: graphite: (pid 61) 5s
run: nginx: (pid 51) 5s
run: redis: (pid 66) 5s
...
```

At the same I see data inside redis:
```
% docker exec -it 616f00db971e redis-cli --stat
------- data ------ --------------------- load -------------------- - child -
keys       mem      clients blocked requests            connections
182        1.02M    5       0       768 (+0)            9
182        1.02M    5       0       769 (+1)            9
182        1.02M    5       0       770 (+1)            9
182        1.02M    5       0       779 (+9)            9
182        1.02M    5       0       780 (+1)            9
182        1.02M    5       0       781 (+1)            9
182        1.02M    5       0       782 (+1)            9
182        1.02M    5       0       783 (+1)            9
182        1.02M    5       0       784 (+1)            9
```
```

**Comment by deniszh** [comment]
Ah, sorry, I read it wrong.

**Comment by deniszh** [comment]
Yes, `/var/log/redis/` should be created automatically.

**Comment by deniszh** [comment]
Fixed in 1.1.6-1
